ale/drivers/mro_drivers.py

```diff
@@ -196,3 +197,7 @@ def detector_start_sample(self):
     @property
     def sensor_model_version(self):
         return 1
+
+    @property
+    def exposure_duration(self):
+        return self.line_exposure_duration
```

Comment:
Would it be worth while to only have an exposure_duration property in the label mixins that checks whether the label has line exposure duration or just exposure duration and returns whichever one it has? @Kelvinrr brought this up to me a while ago, seems like it might be nice to make things a little simpler.